### DISCUSSION TITLE
[KeePassXC-Browser] I can connect to websites without the connection with this plugin. Is this insecure?

### DISCUSSION THREAD

**Discussion by 7arma7** [keepassxreboot/keepassxc#9856] [Q&A] [answered]
I saved a mailbox password in the KeepassXC database. Then I installed KeepassXc browser plugin for Chrome to automatically fill in the password. KeepassXc is unlocked and connected to the plugin.
It works very well.

But, I noticed that I could connect to my mailbox even if KeepassXC was locked and even without launching the application.
The browser says: The database is not open or not connected.

But I was able to connect to my mailbox without filling in the password even though KeepassXC was disconnected.

Can I continue to connect to my mailbox in this way?

**Comment by droidmonkey** [comment, marked as answer]
I think you are referring to being signed in via a saved cookie. That is a popular way to use the internet, yes, it is also called "cached logon". We aren't involved in that at all, it's between you, the website and your browser.

**Comment by 7arma7** [comment]
Hello droïdesinge, thank you for your reply,

Where does this cookie come from? KeppassXc browser plugin?

**Reply by droidmonkey** [reply]
The website
